cdap-docs/developers-manual/source/advanced/index.rst

```diff
@@ -23,6 +24,11 @@ will be of interest to developers who want a deeper dive into CDAP:
 .. |best-practices| replace:: **Best Practices:**
 .. _best-practices: best-practices.html
 
+.. |adapters| replace:: **Adapters:**
+.. _adapters: adapters.html
+
 - |data-exploration|_ Exploration of CDAP Datasets using SQL.
 
 - |best-practices|_ Suggestions when developing a CDAP application.
+
+- |adapters|_ Adapters connect a data source to a data sink. CDAP currently provides a stream conversion adapter that regularly reads data from a Stream and writes it to a TimePartitionedFileSet, allowing it to be queried through Hive and Impala.
```

Comment:
This needs to be wrapped so that there are leading spaces on the second and third lines. Otherwise the bullet won't work very well. Github is more tolerant than Sphinx. "adapter" -> "Adapter", and change to ``TimePartitionedFileSet`` ?